src/Symfony/Component/ClassLoader/composer.json

```diff
@@ -22,6 +22,9 @@
     "require-dev": {
         "symfony/finder": "~2.7|~3.0"
     },
+    "suggest": {
+        "symfony/debug": "Provides a special DebugClassLoader class loader for debugging purposes."
```

Comment:
This should be removed, because the class-loader component will not do anything with the symfony/debug component when it is available. Both packages are totally independent.
So this suggestion is only adding more output at the end of the composer installation.